app/javascript/dashboard/routes/dashboard/settings/reports/InboxReports.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <woot-reports
+    key="inbox-reports"
+    type="inbox"
+    getter-key="inboxes/getInboxes"
+    action-key="inboxes/get"
+    :download-button-label="$t('REPORT.DOWNLOAD_AGENT_REPORTS')"
```

Comment:
Button text should be "Download inbox reports"

Reply:
Fixed

https://github.com/chatwoot/chatwoot/pull/3084/commits/26ee689277da85b14e4706210103aef7bcb51d06